files/default/phpcheck

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+SEARCHDIR=$(pwd)
```

Comment:
Why can't this just be ``/var/www`` by default?

Reply:
I prefer to `cd` to the directory I work in, and needing to specify the current directory with `-d ` seems redundant. I can have it default to that, but I would prefer not to. I thought that the -d option was more for kitchen than the intended usage.

Reply:
There's nothing stopping you to add ``cd ${SEARCHDIR}`` right before the find command which should replicate the same behaviour.